app/src/organisms/Devices/ProtocolRun/ProtocolDropTipBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import {
+  Flex,
+  SPACING,
+  Btn,
+  TYPOGRAPHY,
+  ALIGN_START,
+  DIRECTION_COLUMN,
+  DIRECTION_ROW,
+  FONT_WEIGHT_SEMIBOLD,
+} from '@opentrons/components'
+
+import { Banner } from '../../../atoms/Banner'
+import { StyledText } from '../../../atoms/text'
+
+export function ProtocolDropTipBanner(props: {
+  onLaunchWizardClick: (setShowWizard: true) => void
+  onCloseClick: () => void
+}): JSX.Element {
+  const { t } = useTranslation('drop_tip_wizard')
+  const { onLaunchWizardClick, onCloseClick } = props
+
+  return (
+    <Banner
+      type="warning"
+      onCloseClick={onCloseClick}
+      marginBottom={SPACING.spacing16}
+      paddingRight={SPACING.spacing16}
+    >
+      <Flex flexDirection={DIRECTION_COLUMN}>
+        <StyledText as="p" fontWeight={FONT_WEIGHT_SEMIBOLD}>
```

Comment:
we can use `TYPOGRAPHY.fontWeightSemiBold` instead of `FONT_WEIGHT_SEMIBOLD`

```suggestion
        <StyledText as="p" fontWeight={TYPOGRAPHY.fontWeightSemiBold}>
```